test/data/note

```diff
@@ -0,0 +1 @@
+{"id":123456,"content":"test test test"}
```

Comment:
This sort of thing should be generated within your test file.  Since it is data held within the test folder I won't consider it unnecessary, but generally it is better to have the data you use in your tests created inside the test files themselves and then have that same data deleted within the test file at the end of your tests.